test/kongintegration/translator_golden_tests_outputs_test.go

```diff
@@ -108,35 +99,47 @@ func TestTranslatorsGoldenTestsOutputs_Konnect(t *testing.T) {
 
 	for _, goldenTestOutputPath := range allGoldenTestsOutputsPaths(t) {
 		t.Run(goldenTestOutputPath, func(t *testing.T) {
-			ensureGoldenTestOutputIsAccepted(ctx, t, goldenTestOutputPath, updateStrategy)
+			goldenTestOutput, err := os.ReadFile(goldenTestOutputPath)
+			require.NoError(t, err)
+
+			content := &file.Content{}
+			err = yaml.Unmarshal(goldenTestOutput, content)
+			require.NoError(t, err)
+
+			require.Eventually(t, func() bool {
+				if err := updateStrategy.Update(ctx, sendconfig.ContentWithHash{Content: content}); err != nil {
+					t.Logf("error: %v", err)
+					return false
+				}
+				return true
+			}, timeout, tick)
```

Comment:
Maybe it is better to use `require.EventuallyWithT` to not spam with some redundant logs, WDYT?